### DISCUSSION TITLE
Cant find the GoogleAnalytics function in @next/third-parties/google

### DISCUSSION THREAD

**Discussion by amosmachora** [vercel/next.js#59912] [Help]
### Summary

I don't know if it's just me but I get the error `Module '"@next/third-parties/google"' has no exported member 'GoogleAnalytics'.ts(2305)` The only exported Google functions i get are 

`export { default as GoogleMapsEmbed } from './google-maps-embed';
export { default as YouTubeEmbed } from './youtube-embed';
export { GoogleTagManager, sendGTMEvent } from './gtm';
`
Have I installed the wrong package? I have been following the documentation to the letter. A similar issue at #58697 has been fixed for GoogleTagManager, not for Google Analytics, or can they be used interchangeably? 

Someone help.

**Comment by codeBelt** [comment]
I have the same issue. You might want to check out this thread https://github.com/vercel/next.js/issues/57643

**Comment by mrksph** [comment]
Yeah, still facing the same issue @housseindjirde
